backends/xnnpack/test/tester/tester.py

```diff
@@ -27,9 +29,16 @@
 from executorch.exir.passes.spec_prop_pass import SpecPropPass
 from executorch.exir.print_program import pretty_print, print_program
 
-from executorch.extension.pybindings.portable_lib import (  # @manual
-    _load_for_executorch_from_buffer,
-)
+logger = logging.getLogger(__name__)
```

Comment:
this is nice, I was thinking about adding something, can we override the `level` through a `verbosity` arg in the Tester.__init__()?

Reply:
Will look into it. Not sure what a verbosity args is just yet :)